.travis.yml

```diff
@@ -27,18 +34,18 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
   # Create conda test environment
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy pandas xarray requests beautifulsoup4 lxml netCDF4 pytest-cov pytest-ordering future
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
   - conda activate test-environment
+  # Once pysat 3.0 is released, remove the custom pysat install and change
+  # this to:
+  # - while read requirement; do conda install --yes $requirement; done < requirements.txt1
+  - conda install --yes numpy scipy pandas xarray requests netCDF4
   # Get latest coveralls from pip, not conda
-  - pip install coveralls
-  - pip install pytest-flake8
-  # Dependencies not available through conda, install through pip
-  - pip install pysatCDF >/dev/null
+  - pip install -rtest_requirements.txt
```

Comment:
```suggestion
  - pip install -r test_requirements.txt
```

Reply:
It works either way, but it is nicer with whitespace.